test/integration/contract.js

```diff
@@ -215,12 +215,11 @@ describe('Contract', function () {
 
       // preclaim
       const { salt: _salt } = await contract.aensPreclaim(name)
-      // @TODO enable after next HF
-      // const commitmentId = commitmentHash(name, _salt)
+      const commitmentId = commitmentHash(name, _salt)
```

Comment:
the salt used here needs to be generated with the utils independent from the `aensPreclaim` method